### ISSUE TITLE
About Data Split

### ISSUE THREAD

**Issue by NamecantbeNULL** [ChenyangLEI/CVPR2020-Polarized-Reflection-Removal-with-Perfect-Alignment#1]
Hi, 
Thank you for your test code and pre-trained models firstly. I am confused about data split in your provided M-R dataset. It seems that the whole dataset consist of 889 image pairs rather than 807 in README. And which images are belong to testing set or validation set?
Looking forward to your reply :-)

**Comment by ChenyangLEI** [comment]
Hi, we collect more image compared with the submission version. To previde as much data as possible, we upload all of them. For the val/ test split, do you mean the 107 test set in our paper? If you need it, we can sent a .txt to you to your email and we will update this repo later.

**Comment by NamecantbeNULL** [comment]
That's fine. Yes, I need that TXT file containing the list of val and test set. Thanks a lot. And my email address is doublebin111@163.com
Thank you for your reply.

**Comment by ChenyangLEI** [comment]
[train.txt](https://github.com/ChenyangLEI/CVPR2020-Polarized-Reflection-Removal-with-Perfect-Alignment/files/5265217/train.txt)
[val.txt](https://github.com/ChenyangLEI/CVPR2020-Polarized-Reflection-Removal-with-Perfect-Alignment/files/5265218/val.txt)
[test.txt](https://github.com/ChenyangLEI/CVPR2020-Polarized-Reflection-Removal-with-Perfect-Alignment/files/5265219/test.txt)

The extra files are added in 'val' set. The number of train/ test set should be consistent with the split in our paper.
